Minio/DefaultRequestLogger.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * Minio .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Text;
+using Minio.DataModel.Tracing;
+
+namespace Minio
+{
+    internal sealed class DefaultRequestLogger : IRequestLogger
```

Comment:
@SanSYS, since SetTraceOn() signature is changing, make this class public, and update examples/documentation to use the new signature. API.md and Minio.Examples/Program.cs need to change